docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_general.scss

```diff
@@ -69,8 +69,9 @@ h6 {
   margin-top: 40px;
   font-style: normal;
 }
-h2.recommended-content__header {
+h4.recommended-content__heading {
```

Comment:
you made this change in https://github.com/NCIOCPL/cgov-digital-platform/pull/4415/files - are you reverting it ?